app/models/forms/move.rb

```diff
@@ -41,5 +41,34 @@ def not_for_release_reasons
     def not_for_release_reason_with_details
       REASON_WITH_DETAILS
     end
+
+    FREE_FORM_DESTINATION_TYPES = %i[hospital other].freeze
+
+    property :to_type, validates: { presence: true }
+
+    ::Establishment::ESTABLISHMENT_TYPES.each do |establishment_type|
```

Comment:
::Establishment::ESTABLISHMENT_TYPES -> Establishment::ESTABLISHMENT_TYPES

Reply:
Without the leading :: we get `NameError: wrong constant name move` when we run the specs

Reply:
👍